pennylane/variable.py

```diff
@@ -167,16 +167,16 @@ def render(self, show_name_only=False):
         """
         if not show_name_only:
             if self.is_kwarg and Variable.kwarg_values and self.name in Variable.kwarg_values:
-                return str(round(self.val, 3))
+                return f"{self.val:.3g}"
```

Comment:
These were changed to match non-Variable parameter printing. We cannot use `round()` if the value is a TensorFlow tensor, hence the switch to string formatting (which _does_ work for tensorflow).

Reply:
Clever!